src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -411,6 +411,18 @@ private function addHttpCacheSection(ArrayNodeDefinition $rootNode): void
                                     ->end()
                                     ->info('To pass options to the client charged with the request.')
                                 ->end()
+                                ->booleanNode('xkey')
+                                    ->defaultFalse()
+                                    ->info('Enable support for xkey invalidation')
+                                ->end()
+                                ->scalarNode('xkey_glue')
+                                    ->defaultValue(' ')
+                                    ->info('xkey glue between keys')
+                                ->end()
+                                ->booleanNode('http_tags')
+                                    ->defaultTrue()
```

Comment:
It would be better to use `canBeDisabled()`.

Reply:
imo this is overrated as the http tags is only a way of invalidating

Reply:
Yes but using `canBeDisabled()` would allow to add more options in the future.